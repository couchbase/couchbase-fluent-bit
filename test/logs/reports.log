[error_logger:info,2021-03-09T17:31:57.613Z,nonode@nohost:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]
=========================PROGRESS REPORT=========================
          supervisor: {local,ssl_dist_admin_sup}
             started: [{pid,<0.170.0>},
                       {id,ssl_pem_cache_dist},
                       {mfargs,{ssl_pem_cache,start_link_dist,[[]]}},
                       {restart_type,permanent},
                       {shutdown,4000},
                       {child_type,worker}]

[error_logger:info,2021-03-09T17:31:57.637Z,ns_1@cb.local:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]
=========================INFO REPORT=========================
{net_kernel,{connect,normal,'babysitter_of_ns_1@cb.local'}}
[error_logger:info,2021-03-09T17:31:58.392Z,ns_1@cb.local:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]
=========================PROGRESS REPORT=========================
          supervisor: {<0.224.0>,menelaus_web}
             started: [{pid,<0.225.0>},
                       {id,menelaus_web_ipv4},
                       {mfargs,
                        {menelaus_web,http_server,
                         [[{ip,"0.0.0.0"},
                           {name,menelaus_web_ssl_ipv4},
                           {ssl,true},
                           {ssl_opts,
                            [{keyfile,
                              "/opt/couchbase/var/lib/couchbase/config/ssl-cert-key.pem"},
                             {certfile,
                              "/opt/couchbase/var/lib/couchbase/config/ssl-cert-key.pem"},
                             {versions,['tlsv1.1','tlsv1.2']},
                             {cacerts,
                              [<<48,130,3,2,48,130,1,234,160,3,2,1,2,2,8,22,
                                 106,188,231,74,125,50,124,48,13,6,9,42,134,
                                 72,134,247,13,1,1,11,5,0,48,36,49,34,48,32,
                                 6,3,85,4,3,19,25,67,111,117,99,104,98,97,
                                 115,101,32,83,101,114,118,101,114,32,97,56,
                                 102,49,55,51,97,48,48,30,23,13,49,51,48,49,
                                 48,49,48,48,48,48,48,48,90,23,13,52,57,49,
                                 50,51,49,50,51,53,57,53,57,90,48,36,49,34,
                                 48,32,6,3,85,4,3,19,25,67,111,117,99,104,98,
                                 97,115,101,32,83,101,114,118,101,114,32,97,
                                 56,102,49,55,51,97,48,48,130,1,34,48,13,6,9,
                                 42,134,72,134,247,13,1,1,1,5,0,3,130,1,15,0,
                                 48,130,1,10,2,130,1,1,0,214,1,120,202,81,88,
                                 166,180,109,224,198,239,219,134,15,238,27,
                                 205,20,72,119,40,93,50,122,183,45,113,252,
                                 204,32,193,2,101,7,90,77,145,38,132,25,175,
                                 249,254,83,45,22,84,157,104,21,162,79,132,
                                 202,229,6,43,88,135,125,164,245,227,14,18,
                                 31,169,55,191,234,47,43,205,11,242,136,88,
                                 156,63,190,114,244,246,60,192,12,163,187,
                                 178,146,3,31,182,137,66,57,100,7,65,40,50,
                                 48,132,85,173,11,221,169,9,138,249,142,43,
                                 48,154,73,82,159,77,21,117,5,193,184,22,39,
                                 45,134,215,229,247,208,241,201,86,103,169,
                                 106,102,22,82,0,251,7,155,209,230,14,66,136,
                                 27,209,15,62,250,179,68,25,68,37,152,103,
                                 172,155,124,225,59,196,167,207,112,246,234,
                                 210,207,126,90,250,41,250,117,191,124,81,
                                 215,192,249,124,114,123,200,176,188,32,221,
                                 226,11,172,111,104,215,93,95,243,120,132,29,
                                 189,216,54,23,119,42,133,228,189,3,62,24,
                                 163,13,122,81,103,137,189,157,86,16,157,181,
                                 68,108,181,36,107,34,201,111,73,195,20,23,
                                 129,105,12,60,1,202,97,30,12,221,178,177,2,
                                 3,1,0,1,163,56,48,54,48,14,6,3,85,29,15,1,1,
                                 255,4,4,3,2,2,164,48,19,6,3,85,29,37,4,12,
                                 48,10,6,8,43,6,1,5,5,7,3,1,48,15,6,3,85,29,
                                 19,1,1,255,4,5,48,3,1,1,255,48,13,6,9,42,
                                 134,72,134,247,13,1,1,11,5,0,3,130,1,1,0,
                                 100,188,30,118,197,144,179,211,175,240,19,
                                 196,71,26,42,64,120,107,133,134,78,193,27,
                                 184,121,172,106,88,130,120,214,18,104,75,
                                 244,100,233,189,55,13,138,208,1,129,27,90,
                                 221,45,144,96,90,250,210,16,91,121,245,213,
                                 161,45,37,167,5,114,84,111,23,116,176,170,
                                 139,7,211,90,55,141,52,140,160,224,125,133,
                                 191,231,57,234,184,78,165,116,184,224,105,
                                 79,208,139,38,91,95,115,247,253,100,182,22,
                                 109,71,92,186,76,92,97,255,30,171,162,76,
                                 255,74,72,17,110,232,149,11,126,170,136,254,
                                 12,144,166,228,0,118,63,46,223,207,13,228,
                                 196,238,201,79,7,14,2,79,223,65,219,171,181,
                                 113,47,82,149,84,181,194,30,42,19,218,173,
                                 148,57,0,181,186,83,3,229,180,61,83,26,96,
                                 131,109,120,87,148,247,171,255,142,137,172,
                                 13,128,86,9,251,217,101,130,4,176,57,62,4,
                                 79,113,53,26,189,254,177,241,189,214,23,45,
                                 238,139,222,47,208,99,58,154,37,145,225,227,
                                 253,226,8,209,245,149,139,160,31,210,222,
                                 251,36,167,19,58,180,158,253,126,246,67,234,
                                 223,246,195,0,132,144>>]},
                             {dh,
                              <<48,130,1,8,2,130,1,1,0,152,202,99,248,92,201,
                                35,238,246,5,77,93,120,10,118,129,36,52,111,
                                193,167,220,49,229,106,105,152,133,121,157,73,
                                158,232,153,197,197,21,171,140,30,207,52,165,
                                45,8,221,162,21,199,183,66,211,247,51,224,102,
                                214,190,130,96,253,218,193,35,43,139,145,89,
                                200,250,145,92,50,80,134,135,188,205,254,148,
                                122,136,237,220,186,147,187,104,159,36,147,
                                217,117,74,35,163,145,249,175,242,18,221,124,
                                54,140,16,246,169,84,252,45,47,99,136,30,60,
                                189,203,61,86,225,117,255,4,91,46,110,167,173,
                                106,51,65,10,248,94,225,223,73,40,232,140,26,
                                11,67,170,118,190,67,31,127,233,39,68,88,132,
                                171,224,62,187,207,160,189,209,101,74,8,205,
                                174,146,173,80,105,144,246,25,153,86,36,24,
                                178,163,64,202,221,95,184,110,244,32,226,217,
                                34,55,188,230,55,16,216,247,173,246,139,76,
                                187,66,211,159,17,46,20,18,48,80,27,250,96,
                                189,29,214,234,241,34,69,254,147,103,220,133,
                                40,164,84,8,44,241,61,164,151,9,135,41,60,75,
                                4,202,133,173,72,6,69,167,89,112,174,40,229,
                                171,2,1,2>>},
                             {ciphers,
                              [{ecdhe_ecdsa,aes_256_gcm,aead,sha384},
                               {ecdhe_rsa,aes_256_gcm,aead,sha384},
                               {ecdhe_ecdsa,aes_256_cbc,sha384,sha384},
                               {ecdhe_rsa,aes_256_cbc,sha384,sha384},
                               {ecdh_ecdsa,aes_256_gcm,aead,sha384},
                               {ecdh_rsa,aes_256_gcm,aead,sha384},
                               {ecdh_ecdsa,aes_256_cbc,sha384,sha384},
                               {ecdh_rsa,aes_256_cbc,sha384,sha384},
                               {ecdhe_ecdsa,chacha20_poly1305,aead,sha256},
                               {ecdhe_rsa,chacha20_poly1305,aead,sha256},
                               {dhe_rsa,chacha20_poly1305,aead,sha256},
                               {dhe_rsa,aes_256_gcm,aead,sha384},
                               {dhe_dss,aes_256_gcm,aead,sha384},
                               {dhe_rsa,aes_256_cbc,sha256},
                               {dhe_dss,aes_256_cbc,sha256},
                               {rsa,aes_256_gcm,aead,sha384},
                               {rsa,aes_256_cbc,sha256},
                               {ecdhe_ecdsa,aes_128_gcm,aead,sha256},
                               {ecdhe_rsa,aes_128_gcm,aead,sha256},
                               {ecdhe_ecdsa,aes_128_cbc,sha256,sha256},
                               {ecdhe_rsa,aes_128_cbc,sha256,sha256},
                               {ecdh_ecdsa,aes_128_gcm,aead,sha256},
                               {ecdh_rsa,aes_128_gcm,aead,sha256},
                               {ecdh_ecdsa,aes_128_cbc,sha256,sha256},
                               {ecdh_rsa,aes_128_cbc,sha256,sha256},
                               {dhe_rsa,aes_128_gcm,aead,sha256},
                               {dhe_dss,aes_128_gcm,aead,sha256},
                               {dhe_rsa,aes_128_cbc,sha256},
                               {dhe_dss,aes_128_cbc,sha256},
                               {rsa,aes_128_gcm,aead,sha256},
                               {rsa,aes_128_cbc,sha256},
                               {ecdhe_ecdsa,aes_256_cbc,sha},
                               {ecdhe_rsa,aes_256_cbc,sha},
                               {dhe_rsa,aes_256_cbc,sha},
                               {dhe_dss,aes_256_cbc,sha},
                               {ecdh_ecdsa,aes_256_cbc,sha},
                               {ecdh_rsa,aes_256_cbc,sha},
                               {rsa,aes_256_cbc,sha},
                               {ecdhe_ecdsa,aes_128_cbc,sha},
                               {ecdhe_rsa,aes_128_cbc,sha},
                               {dhe_rsa,aes_128_cbc,sha},
                               {dhe_dss,aes_128_cbc,sha},
                               {ecdh_ecdsa,aes_128_cbc,sha},
                               {ecdh_rsa,aes_128_cbc,sha},
                               {rsa,aes_128_cbc,sha},
                               {ecdhe_ecdsa,'3des_ede_cbc',sha},
                               {ecdhe_rsa,'3des_ede_cbc',sha},
                               {dhe_rsa,'3des_ede_cbc',sha},
                               {dhe_dss,'3des_ede_cbc',sha},
                               {ecdh_ecdsa,'3des_ede_cbc',sha},
                               {ecdh_rsa,'3des_ede_cbc',sha},
                               {rsa,'3des_ede_cbc',sha}]},
                             {honor_cipher_order,true},
                             {secure_renegotiate,true},
                             {client_renegotiation,false}]},
                           {port,18091}]]}},
                       {restart_type,permanent},
                       {shutdown,5000},
                       {child_type,worker}]

[error_logger:info,2021-03-09T17:31:58.458Z,ns_1@cb.local:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]
=========================INFO REPORT=========================
{net_kernel,{connect,normal,'couchdb_ns_1@cb.local'}}
[error_logger:info,2021-03-09T17:32:14.393Z,ns_1@cb-example-0000.cb-example.default.svc:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]
=========================INFO REPORT=========================
{net_kernel,{net_kernel,913,nodedown,
                        'ns_1@cb-example-0000.cb-example.default.svc'}}
[error_logger:error,2021-03-09T17:32:14.418Z,ns_1@cb-example-0000.cb-example.default.svc:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]** Generic server leader_quorum_nodes_manager terminating 
** Last message in was quorum_nodes_updated
** When Server state == {state,{set,1,16,16,8,80,48,
                                    {[],[],[],[],[],[],[],[],[],[],[],[],[],
                                     [],[],[]},
                                    {{[],[],[],[],[],[],[],[],[],[],[],[],
                                      ['ns_1@cb.local'],
                                      [],[],[]}}}}
** Reason for termination == 
** {{quorum_nodes_update_conflict,
        ['ns_1@cb.local'],
        ['ns_1@cb-example-0000.cb-example.default.svc']},
    [{leader_quorum_nodes_manager,handle_quorum_nodes_updated,1,
         [{file,"src/leader_quorum_nodes_manager.erl"},{line,175}]},
     {leader_quorum_nodes_manager,handle_info,2,
         [{file,"src/leader_quorum_nodes_manager.erl"},{line,95}]},
     {gen_server2,handle_info,2,[{file,"src/gen_server2.erl"},{line,228}]},
     {gen_server,try_dispatch,4,[{file,"gen_server.erl"},{line,616}]},
     {gen_server,handle_msg,6,[{file,"gen_server.erl"},{line,686}]},
     {proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,247}]}]}

[error_logger:error,2021-03-09T17:32:14.437Z,ns_1@cb-example-0000.cb-example.default.svc:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]
=========================CRASH REPORT=========================
  crasher:
    initial call: leader_quorum_nodes_manager:init/1
    pid: <0.561.0>
    registered_name: leader_quorum_nodes_manager
    exception exit: {quorum_nodes_update_conflict,
                        ['ns_1@cb.local'],
                        ['ns_1@cb-example-0000.cb-example.default.svc']}
      in function  leader_quorum_nodes_manager:handle_quorum_nodes_updated/1 (src/leader_quorum_nodes_manager.erl, line 175)
      in call from leader_quorum_nodes_manager:handle_info/2 (src/leader_quorum_nodes_manager.erl, line 95)
      in call from gen_server2:handle_info/2 (src/gen_server2.erl, line 228)
      in call from gen_server:try_dispatch/4 (gen_server.erl, line 616)
      in call from gen_server:handle_msg/6 (gen_server.erl, line 686)
    ancestors: [mb_master_sup,mb_master,leader_registry_sup,
                  leader_services_sup,<0.547.0>,ns_server_sup,
                  ns_server_nodes_sup,<0.206.0>,ns_server_cluster_sup,
                  root_sup,<0.118.0>]
    message_queue_len: 0
    messages: []
    links: [<0.558.0>,<0.578.0>]
    dictionary: [{{'$gen_server2',{have_callback,handle_job_death}},false},
                  {{'$gen_server2',module},leader_quorum_nodes_manager},
                  {{'$gen_server2',{have_callback,code_change}},false},
                  {{'$gen_server2',{have_callback,handle_call}},true},
                  {{'$gen_server2',{have_callback,terminate}},false},
                  {{'$gen_server2',{have_callback,init}},true},
                  {{'$gen_server2',{have_callback,handle_cast}},false},
                  {{'$gen_server2',{have_callback,handle_info}},true}]
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 27
    reductions: 8107
  neighbours:

