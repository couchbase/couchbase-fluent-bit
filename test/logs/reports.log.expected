/fluent-bit/test/logs/reports.log.actual: [1615311117.613000000, {"filename":"/fluent-bit/test/logs/reports.log","logger":"error_logger","timestamp":"2021-03-09T17:31:57.613Z","message":"nonode@nohost:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]\n=========================PROGRESS REPORT=========================\n          supervisor: {local,ssl_dist_admin_sup}\n             started: [{pid,<0.170.0>},\n                       {id,ssl_pem_cache_dist},\n                       {mfargs,{ssl_pem_cache,start_link_dist,[[]]}},\n                       {restart_type,permanent},\n                       {shutdown,4000},\n                       {child_type,worker}]","level":"INFO"}]
/fluent-bit/test/logs/reports.log.actual: [1615311117.637000000, {"filename":"/fluent-bit/test/logs/reports.log","logger":"error_logger","timestamp":"2021-03-09T17:31:57.637Z","message":"ns_1@cb.local:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]\n=========================INFO REPORT=========================\n{net_kernel,{connect,normal,'babysitter_of_ns_1@cb.local'}}","level":"INFO"}]
/fluent-bit/test/logs/reports.log.actual: [1615311118.392000000, {"filename":"/fluent-bit/test/logs/reports.log","logger":"error_logger","timestamp":"2021-03-09T17:31:58.392Z","message":"ns_1@cb.local:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]\n=========================PROGRESS REPORT=========================\n          supervisor: {<0.224.0>,menelaus_web}\n             started: [{pid,<0.225.0>},\n                       {id,menelaus_web_ipv4},\n                       {mfargs,\n                        {menelaus_web,http_server,\n                         [[{ip,\"0.0.0.0\"},\n                           {name,menelaus_web_ssl_ipv4},\n                           {ssl,true},\n                           {ssl_opts,\n                            [{keyfile,\n                              \"/opt/couchbase/var/lib/couchbase/config/ssl-cert-key.pem\"},\n                             {certfile,\n                              \"/opt/couchbase/var/lib/couchbase/config/ssl-cert-key.pem\"},\n                             {versions,['tlsv1.1','tlsv1.2']},\n                             {cacerts,\n                              [<<48,130,3,2,48,130,1,234,160,3,2,1,2,2,8,22,\n                                 106,188,231,74,125,50,124,48,13,6,9,42,134,\n                                 72,134,247,13,1,1,11,5,0,48,36,49,34,48,32,\n                                 6,3,85,4,3,19,25,67,111,117,99,104,98,97,\n                                 115,101,32,83,101,114,118,101,114,32,97,56,\n                                 102,49,55,51,97,48,48,30,23,13,49,51,48,49,\n                                 48,49,48,48,48,48,48,48,90,23,13,52,57,49,\n                                 50,51,49,50,51,53,57,53,57,90,48,36,49,34,\n                                 48,32,6,3,85,4,3,19,25,67,111,117,99,104,98,\n                                 97,115,101,32,83,101,114,118,101,114,32,97,\n                                 56,102,49,55,51,97,48,48,130,1,34,48,13,6,9,\n                                 42,134,72,134,247,13,1,1,1,5,0,3,130,1,15,0,\n                                 48,130,1,10,2,130,1,1,0,214,1,120,202,81,88,\n                                 166,180,109,224,198,239,219,134,15,238,27,\n                                 205,20,72,119,40,93,50,122,183,45,113,252,\n                                 204,32,193,2,101,7,90,77,145,38,132,25,175,\n                                 249,254,83,45,22,84,157,104,21,162,79,132,\n                                 202,229,6,43,88,135,125,164,245,227,14,18,\n                                 31,169,55,191,234,47,43,205,11,242,136,88,\n                                 156,63,190,114,244,246,60,192,12,163,187,\n                                 178,146,3,31,182,137,66,57,100,7,65,40,50,\n                                 48,132,85,173,11,221,169,9,138,249,142,43,\n                                 48,154,73,82,159,77,21,117,5,193,184,22,39,\n                                 45,134,215,229,247,208,241,201,86,103,169,\n                                 106,102,22,82,0,251,7,155,209,230,14,66,136,\n                                 27,209,15,62,250,179,68,25,68,37,152,103,\n                                 172,155,124,225,59,196,167,207,112,246,234,\n                                 210,207,126,90,250,41,250,117,191,124,81,\n                                 215,192,249,124,114,123,200,176,188,32,221,\n                                 226,11,172,111,104,215,93,95,243,120,132,29,\n                                 189,216,54,23,119,42,133,228,189,3,62,24,\n                                 163,13,122,81,103,137,189,157,86,16,157,181,\n                                 68,108,181,36,107,34,201,111,73,195,20,23,\n                                 129,105,12,60,1,202,97,30,12,221,178,177,2,\n                                 3,1,0,1,163,56,48,54,48,14,6,3,85,29,15,1,1,\n                                 255,4,4,3,2,2,164,48,19,6,3,85,29,37,4,12,\n                                 48,10,6,8,43,6,1,5,5,7,3,1,48,15,6,3,85,29,\n                                 19,1,1,255,4,5,48,3,1,1,255,48,13,6,9,42,\n                                 134,72,134,247,13,1,1,11,5,0,3,130,1,1,0,\n                                 100,188,30,118,197,144,179,211,175,240,19,\n                                 196,71,26,42,64,120,107,133,134,78,193,27,\n                                 184,121,172,106,88,130,120,214,18,104,75,\n                                 244,100,233,189,55,13,138,208,1,129,27,90,\n                                 221,45,144,96,90,250,210,16,91,121,245,213,\n                                 161,45,37,167,5,114,84,111,23,116,176,170,\n                                 139,7,211,90,55,141,52,140,160,224,125,133,\n                                 191,231,57,234,184,78,165,116,184,224,105,\n                                 79,208,139,38,91,95,115,247,253,100,182,22,\n                                 109,71,92,186,76,92,97,255,30,171,162,76,\n                                 255,74,72,17,110,232,149,11,126,170,136,254,\n                                 12,144,166,228,0,118,63,46,223,207,13,228,\n                                 196,238,201,79,7,14,2,79,223,65,219,171,181,\n                                 113,47,82,149,84,181,194,30,42,19,218,173,\n                                 148,57,0,181,186,83,3,229,180,61,83,26,96,\n                                 131,109,120,87,148,247,171,255,142,137,172,\n                                 13,128,86,9,251,217,101,130,4,176,57,62,4,\n                                 79,113,53,26,189,254,177,241,189,214,23,45,\n                                 238,139,222,47,208,99,58,154,37,145,225,227,\n                                 253,226,8,209,245,149,139,160,31,210,222,\n                                 251,36,167,19,58,180,158,253,126,246,67,234,\n                                 223,246,195,0,132,144>>]},\n                             {dh,\n                              <<48,130,1,8,2,130,1,1,0,152,202,99,248,92,201,\n                                35,238,246,5,77,93,120,10,118,129,36,52,111,\n                                193,167,220,49,229,106,105,152,133,121,157,73,\n                                158,232,153,197,197,21,171,140,30,207,52,165,\n                                45,8,221,162,21,199,183,66,211,247,51,224,102,\n                                214,190,130,96,253,218,193,35,43,139,145,89,\n                                200,250,145,92,50,80,134,135,188,205,254,148,\n                                122,136,237,220,186,147,187,104,159,36,147,\n                                217,117,74,35,163,145,249,175,242,18,221,124,\n                                54,140,16,246,169,84,252,45,47,99,136,30,60,\n                                189,203,61,86,225,117,255,4,91,46,110,167,173,\n                                106,51,65,10,248,94,225,223,73,40,232,140,26,\n                                11,67,170,118,190,67,31,127,233,39,68,88,132,\n                                171,224,62,187,207,160,189,209,101,74,8,205,\n                                174,146,173,80,105,144,246,25,153,86,36,24,\n                                178,163,64,202,221,95,184,110,244,32,226,217,\n                                34,55,188,230,55,16,216,247,173,246,139,76,\n                                187,66,211,159,17,46,20,18,48,80,27,250,96,\n                                189,29,214,234,241,34,69,254,147,103,220,133,\n                                40,164,84,8,44,241,61,164,151,9,135,41,60,75,\n                                4,202,133,173,72,6,69,167,89,112,174,40,229,\n                                171,2,1,2>>},\n                             {ciphers,\n                              [{ecdhe_ecdsa,aes_256_gcm,aead,sha384},\n                               {ecdhe_rsa,aes_256_gcm,aead,sha384},\n                               {ecdhe_ecdsa,aes_256_cbc,sha384,sha384},\n                               {ecdhe_rsa,aes_256_cbc,sha384,sha384},\n                               {ecdh_ecdsa,aes_256_gcm,aead,sha384},\n                               {ecdh_rsa,aes_256_gcm,aead,sha384},\n                               {ecdh_ecdsa,aes_256_cbc,sha384,sha384},\n                               {ecdh_rsa,aes_256_cbc,sha384,sha384},\n                               {ecdhe_ecdsa,chacha20_poly1305,aead,sha256},\n                               {ecdhe_rsa,chacha20_poly1305,aead,sha256},\n                               {dhe_rsa,chacha20_poly1305,aead,sha256},\n                               {dhe_rsa,aes_256_gcm,aead,sha384},\n                               {dhe_dss,aes_256_gcm,aead,sha384},\n                               {dhe_rsa,aes_256_cbc,sha256},\n                               {dhe_dss,aes_256_cbc,sha256},\n                               {rsa,aes_256_gcm,aead,sha384},\n                               {rsa,aes_256_cbc,sha256},\n                               {ecdhe_ecdsa,aes_128_gcm,aead,sha256},\n                               {ecdhe_rsa,aes_128_gcm,aead,sha256},\n                               {ecdhe_ecdsa,aes_128_cbc,sha256,sha256},\n                               {ecdhe_rsa,aes_128_cbc,sha256,sha256},\n                               {ecdh_ecdsa,aes_128_gcm,aead,sha256},\n                               {ecdh_rsa,aes_128_gcm,aead,sha256},\n                               {ecdh_ecdsa,aes_128_cbc,sha256,sha256},\n                               {ecdh_rsa,aes_128_cbc,sha256,sha256},\n                               {dhe_rsa,aes_128_gcm,aead,sha256},\n                               {dhe_dss,aes_128_gcm,aead,sha256},\n                               {dhe_rsa,aes_128_cbc,sha256},\n                               {dhe_dss,aes_128_cbc,sha256},\n                               {rsa,aes_128_gcm,aead,sha256},\n                               {rsa,aes_128_cbc,sha256},\n                               {ecdhe_ecdsa,aes_256_cbc,sha},\n                               {ecdhe_rsa,aes_256_cbc,sha},\n                               {dhe_rsa,aes_256_cbc,sha},\n                               {dhe_dss,aes_256_cbc,sha},\n                               {ecdh_ecdsa,aes_256_cbc,sha},\n                               {ecdh_rsa,aes_256_cbc,sha},\n                               {rsa,aes_256_cbc,sha},\n                               {ecdhe_ecdsa,aes_128_cbc,sha},\n                               {ecdhe_rsa,aes_128_cbc,sha},\n                               {dhe_rsa,aes_128_cbc,sha},\n                               {dhe_dss,aes_128_cbc,sha},\n                               {ecdh_ecdsa,aes_128_cbc,sha},\n                               {ecdh_rsa,aes_128_cbc,sha},\n                               {rsa,aes_128_cbc,sha},\n                               {ecdhe_ecdsa,'3des_ede_cbc',sha},\n                               {ecdhe_rsa,'3des_ede_cbc',sha},\n                               {dhe_rsa,'3des_ede_cbc',sha},\n                               {dhe_dss,'3des_ede_cbc',sha},\n                               {ecdh_ecdsa,'3des_ede_cbc',sha},\n                               {ecdh_rsa,'3des_ede_cbc',sha},\n                               {rsa,'3des_ede_cbc',sha}]},\n                             {honor_cipher_order,true},\n                             {secure_renegotiate,true},\n                             {client_renegotiation,false}]},\n                           {port,18091}]]}},\n                       {restart_type,permanent},\n                       {shutdown,5000},\n                       {child_type,worker}]","level":"INFO"}]
/fluent-bit/test/logs/reports.log.actual: [1615311118.458000000, {"filename":"/fluent-bit/test/logs/reports.log","logger":"error_logger","timestamp":"2021-03-09T17:31:58.458Z","message":"ns_1@cb.local:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]\n=========================INFO REPORT=========================\n{net_kernel,{connect,normal,'couchdb_ns_1@cb.local'}}","level":"INFO"}]
/fluent-bit/test/logs/reports.log.actual: [1615311134.393000000, {"filename":"/fluent-bit/test/logs/reports.log","logger":"error_logger","timestamp":"2021-03-09T17:32:14.393Z","message":"ns_1@cb-example-0000.cb-example.default.svc:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]\n=========================INFO REPORT=========================\n{net_kernel,{net_kernel,913,nodedown,\n                        'ns_1@cb-example-0000.cb-example.default.svc'}}","level":"INFO"}]
/fluent-bit/test/logs/reports.log.actual: [1615311134.418000000, {"filename":"/fluent-bit/test/logs/reports.log","logger":"error_logger","timestamp":"2021-03-09T17:32:14.418Z","message":"ns_1@cb-example-0000.cb-example.default.svc:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]** Generic server leader_quorum_nodes_manager terminating \n** Last message in was quorum_nodes_updated\n** When Server state == {state,{set,1,16,16,8,80,48,\n                                    {[],[],[],[],[],[],[],[],[],[],[],[],[],\n                                     [],[],[]},\n                                    {{[],[],[],[],[],[],[],[],[],[],[],[],\n                                      ['ns_1@cb.local'],\n                                      [],[],[]}}}}\n** Reason for termination == \n** {{quorum_nodes_update_conflict,\n        ['ns_1@cb.local'],\n        ['ns_1@cb-example-0000.cb-example.default.svc']},\n    [{leader_quorum_nodes_manager,handle_quorum_nodes_updated,1,\n         [{file,\"src/leader_quorum_nodes_manager.erl\"},{line,175}]},\n     {leader_quorum_nodes_manager,handle_info,2,\n         [{file,\"src/leader_quorum_nodes_manager.erl\"},{line,95}]},\n     {gen_server2,handle_info,2,[{file,\"src/gen_server2.erl\"},{line,228}]},\n     {gen_server,try_dispatch,4,[{file,\"gen_server.erl\"},{line,616}]},\n     {gen_server,handle_msg,6,[{file,\"gen_server.erl\"},{line,686}]},\n     {proc_lib,init_p_do_apply,3,[{file,\"proc_lib.erl\"},{line,247}]}]}","level":"ERROR"}]
/fluent-bit/test/logs/reports.log.actual: [1615311134.437000000, {"filename":"/fluent-bit/test/logs/reports.log","logger":"error_logger","timestamp":"2021-03-09T17:32:14.437Z","message":"ns_1@cb-example-0000.cb-example.default.svc:error_logger<0.32.0>:ale_error_logger_handler:do_log:203]\n=========================CRASH REPORT=========================\n  crasher:\n    initial call: leader_quorum_nodes_manager:init/1\n    pid: <0.561.0>\n    registered_name: leader_quorum_nodes_manager\n    exception exit: {quorum_nodes_update_conflict,\n                        ['ns_1@cb.local'],\n                        ['ns_1@cb-example-0000.cb-example.default.svc']}\n      in function  leader_quorum_nodes_manager:handle_quorum_nodes_updated/1 (src/leader_quorum_nodes_manager.erl, line 175)\n      in call from leader_quorum_nodes_manager:handle_info/2 (src/leader_quorum_nodes_manager.erl, line 95)\n      in call from gen_server2:handle_info/2 (src/gen_server2.erl, line 228)\n      in call from gen_server:try_dispatch/4 (gen_server.erl, line 616)\n      in call from gen_server:handle_msg/6 (gen_server.erl, line 686)\n    ancestors: [mb_master_sup,mb_master,leader_registry_sup,\n                  leader_services_sup,<0.547.0>,ns_server_sup,\n                  ns_server_nodes_sup,<0.206.0>,ns_server_cluster_sup,\n                  root_sup,<0.118.0>]\n    message_queue_len: 0\n    messages: []\n    links: [<0.558.0>,<0.578.0>]\n    dictionary: [{{'$gen_server2',{have_callback,handle_job_death}},false},\n                  {{'$gen_server2',module},leader_quorum_nodes_manager},\n                  {{'$gen_server2',{have_callback,code_change}},false},\n                  {{'$gen_server2',{have_callback,handle_call}},true},\n                  {{'$gen_server2',{have_callback,terminate}},false},\n                  {{'$gen_server2',{have_callback,init}},true},\n                  {{'$gen_server2',{have_callback,handle_cast}},false},\n                  {{'$gen_server2',{have_callback,handle_info}},true}]\n    trap_exit: true\n    status: running\n    heap_size: 2586\n    stack_size: 27\n    reductions: 8107\n  neighbours:","level":"ERROR"}]
